src/app/Todo/Todo.tsx

```
@@ -1,15 +1,58 @@
 "use client";
 
+import { IconX } from "@tabler/icons-react";
+
 export type TodoItem = {
   id: number;
   text: string;
   completed: boolean;
 };
 
-export const Todo = ({ id, text, completed }: TodoItem) => {
+interface TodoProps extends TodoItem {
+  onToggle: () => void;
+  onDelete: () => void;
+  disabled?: boolean;
+}
+
+export const Todo = ({
+  id,
```

If you run `npm run lint` you'll see eslint complains about `id` not being used -- ideally this would have been removed as a prop since it is unused.

---

I saw that and I was thinking about doing _id to ignore the error but I didn't work so I just left it as a warning error